airflow/ui/openapi-gen/requests/types.gen.ts

```diff
@@ -688,6 +688,47 @@ export type FastAPIAppResponse = {
   [key: string]: unknown | string;
 };
 
+/**
+ * DAG Run model for the Grid UI.
+ */
+export type GridDAGRunwithTIs = {
+  run_id: string;
+  queued_at: string | null;
+  start_date: string | null;
+  end_date: string | null;
+  state: string;
+  run_type: string;
+  data_interval_start: string | null;
+  data_interval_end: string | null;
+  version_number: string | null;
+  note: string | null;
+  task_instances: Array<GridTaskInstanceSummary>;
+};
+
+/**
+ * Response model for the Grid UI.
+ */
+export type GridResponse = {
+  dag_runs: Array<GridDAGRunwithTIs>;
+};
+
+/**
+ * Task Instance Summary model for the Grid UI.
+ */
+export type GridTaskInstanceSummary = {
+  task_id: string;
+  try_number: number;
+  start_date: string | null;
+  end_date: string | null;
+  queued_dttm: string | null;
+  states: {
+    [key: string]: number;
+  } | null;
+  task_count: number;
+  overall_state: string | null;
```

Comment:
```suggestion
  state: string | null;
```

I was trying to use this endpoint locally and it is a lot easier if this is closer to `TaskInstanceResponse`

Perhaps we should change `states` to be `child_states` to avoid confusion on variable names?

Reply:
Adjusted accordingly